docs/lcb/notes/20801.md

```diff
@@ -0,0 +1,3 @@
+# LiveCode Builder Standard Library
+## Objective-C Module
```

Comment:
@livecodeali I'm fairly sure unless you actually write a paragraph instead of just headings that the release notes builder won't actually do anything with this when merging the notes together. Bug lines are split out and listed separately. So if you want any new notes merged in under the `Objective-C Module` subheading you need to add it here.

Reply:
Ah ok